### DISCUSSION TITLE
Can I use GO in query command?

### DISCUSSION THREAD

**Discussion by jungjin09** [nanodbc/nanodbc#400] [Q&A]
Hi,
I tried querying with multiple lines but without success.
I have seen this from NET:
```
            Server server = new Server(new ServerConnection(connection));
            server.ConnectionContext.ExecuteNonQuery(script);
```
Is there any other approach?
Thanks you!

**Comment by mloskot** [comment]
From https://learn.microsoft.com/en-us/sql/t-sql/language-elements/sql-server-utilities-statements-go?view=sql-server-ver16

> GO is not a Transact-SQL statement; it is a command recognized by the sqlcmd and osql utilities and SQL Server Management Studio Code editor.



https://learn.microsoft.com/en-us/sql/odbc/reference/develop-app/batches-of-sql-statements
